draft-irtf-cfrg-hpke.md

```diff
@@ -710,11 +711,54 @@ byte strings for public keys.
 | 0x0002 | AES-GCM-256      | 32  | 12  | {{GCM}}      |
 | 0x0003 | ChaCha20Poly1305 | 32  | 12  | {{?RFC8439}} |
 
-# Security Considerations
-
-The general security properties of HPKE are described in
-{{security-properties}}.  In this section, we consider a security issue that may
-arise in practice and an advanced use case.
+# Security Considerations {#sec-considerations}
+
+HPKE makes the following assumptions about its cryptographic
+dependencies.
+
+- Hash: Collision resistance.
+- Extract: Indifferentiable from a Random Oracle.
+- Expand: Behaves as a Pseudorandom Function wherein the first
+argument is the key.
+
+For the DH-based KEM, HPKE also assumes the gap Computational
+Diffie-Hellman (CDH) problem is hard {{S01}}.
+
+HPKE has several security goals, depending on the mode of operation,
+against active and adaptive attackers that can compromise partial secrets
+of senders and recipients. The desired security goals are detailed below:
+
+- Message secrecy: Privacy of the sender's messages, i.e., IND-CCA2
+security.
+- Export key secrecy: Indistinguishability of the export secret from
```

Comment:
export secrets

(could be multiple)

Reply:
Or better:

Indistinguishability of each export secret from a uniformly random bitstring of equal length.

Reply:
👍

Reply:
Nit: "each exported secret"